shell/imports/sandstorm-autoupdate-apps/autoupdate-apps-tests.js

```diff
@@ -23,6 +23,7 @@ import { Meteor } from "meteor/meteor";
 import { Accounts } from "meteor/accounts-base";
 
 import { SandstormDb } from "/imports/sandstorm-db/db.js";
+import { SandstormAutoUpdateApps } from "/imports/sandstorm-autoupdate-apps/autoupdate-apps.js";
```

Comment:
`SandstormAutoUpdateApps` should be `SandstormAutoupdateApps` (the U in update should be lowercase).